frontend/src/features/admin-form/create/builder-and-design/utils/fieldCreation.ts

```diff
@@ -9,9 +9,9 @@ import {
   RatingShape,
 } from '~shared/types/field'
 import {
-  getMyInfoDropdownFieldMeta,
   MYINFO_FIELD_CONSTANTS,
   myInfoDateFieldMeta,
+  myInfoDropdownFieldMeta,
   myInfoMobileFieldMeta,
   myInfoTextFieldMeta,
```

Comment:
i suspect these meta constants do not actually need to be shared. should we move them into the client side? 
nit: should also be in UPPER_CASE

Reply:
good point, changed and relocated them!